OracleTuxedo/performance_pack/dockerfiles/perfpack_runme.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
+#
+# Shell script to build and run perfpack.  It assume TUXDIR has been set, or that 
+# Tuxedo has been installed to: ~/tuxHome/tuxedo12.1.3.0.0   If not, invoke 
+# this script with a single argument indicating the location of TUXDIR.
+#
+# Author: Judy Liu
+#
+# Usage: source perfpack_runme.sh
+#
+cd /u01/oracle/user_projects
+rm -rf perfpack
```

Comment:
If you delete the entire `perfpack` directory on startup, wouldn't this make it impossible to start/stop the container while retaining data? Or is this directory only for ephemeral data?